modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/regions/IT.json

```diff
@@ -64,7 +64,7 @@
 		"regionCode": "BO"
 	},
 	{
-		"name": "Bolzano",
+		"name": "Bozen",
```

Comment:
@wanderlast this is the same example as [here](https://github.com/liferay-commerce/liferay-portal/pull/4359/#discussion_r1391012400). The name is in German but this should be in Italian.